Enterprise-CMCS/managed-care-review
# Analytics bugfix

## Summary
- Get production tealium upgraded to new hostname now that they are ready for us
- Address random issues in [this ticket](https://jira.cms.gov/browse/BLSTANALYT-7663) as much as possible. 
- Turn console logging in dev so we don't flood cypress console. 
- moved types to their own file
- stopped bothering with section subsection and component type since I found out its required asked that this be excluded from our build

#### Related issues
https://jira.cms.gov/browse/BLSTANALYT-7663
https://jiraent.cms.gov/browse/MCR-4301

#### Screenshots
n/a

#### Test cases covered
n/a

## QA guidance
Things I added logging around that should show up locally
- accordion open and close logs something (its accordion_opened always)
-  form submit logs  (just did main form for now, ran out of steam)
- filters pass all the the various filters set with their values
- a bunch of other events have `link_type` set and pass user role and unique identifier (no user data like emails should be in logs)
